pkg/kv/postgres/store_test.go

```diff
@@ -2,29 +2,45 @@ package postgres_test
 
 import (
 	"context"
-	"github.com/treeverse/lakefs/pkg/kv"
-	"github.com/treeverse/lakefs/pkg/testutil"
+	"fmt"
 	"testing"
 
+	"github.com/jackc/pgx/v5"
+	"github.com/treeverse/lakefs/pkg/kv"
 	"github.com/treeverse/lakefs/pkg/kv/kvparams"
 	"github.com/treeverse/lakefs/pkg/kv/kvtest"
 	"github.com/treeverse/lakefs/pkg/kv/postgres"
+	"github.com/treeverse/lakefs/pkg/testutil"
 )
 
 func TestPostgresKV(t *testing.T) {
+	databaseURI, cleanup := runDBInstance(pool, testutil.UniqueKVTableName())
+	t.Cleanup(cleanup)
+
 	kvtest.DriverTest(t, func(t testing.TB, ctx context.Context) kv.Store {
 		t.Helper()
-		databaseURI, cleanup := runDBInstance(pool, testutil.UniqueKVTableName())
+
+		conn, err := pgx.Connect(ctx, databaseURI)
+		if err != nil {
+			t.Fatalf("Unable to connect to database: %v", err)
+		}
+		defer conn.Close(context.Background())
+
+		// create a new schema per test
+		schemaName := "test_schema" + testutil.UniqueName()
+		_, err = conn.Exec(context.Background(), fmt.Sprintf("CREATE SCHEMA IF NOT EXISTS %s;", schemaName))
```

Comment:
Use connection's `QuoteIdentifier` to quote the `schemaName` value.
Check the following if it works:
```suggestion
		_, err = conn.Exec(ctx, "CREATE SCHEMA IF NOT EXISTS " + conn. QuoteIdentifier(schemaName))
```

Reply:
Can't. I simply escaped it